### PR TITLE
rebrand navbar / buttons, some homepage copy tweaks

### PR DESCRIPTION
**Pull request by bee64**
update libraries.io branding

https://github.com/user-attachments/assets/0e381a64-7ee3-43e0-b00b-8e71b1ca2195

### CODE REVIEW

File: app/views/projects/index.html.erb [hunk 1]
```diff
@@ -10,85 +10,97 @@
...
 
+<div class="row">
+  <div class="col-md-4"></div>
+  <% if not logged_in? %>
+    <div class="col-md-2">
+      <%= link_to 'Login', login_path, class:'btn btn-primary center-block' %>
+    </div>
+  <% end %>
+  <div class="col-md-2">
+    <%= link_to 'Documentation', '/api', class:'btn btn-primary center-block'%>
```
Comment (katzj) [inline app/views/projects/index.html.erb:89]:
```suggestion
    <%= link_to 'API Documentation', '/api', class:'btn btn-primary center-block'%>
```

File: app/views/projects/index.html.erb [hunk 2]
```diff
@@ -10,85 +10,97 @@
...
+  </div>
+  <div class="col-md-4"></div>
 </div>
 
 <hr>
 
-<h3 data-ga-tracked-el='search-by-language'>
+<h2>Looking for more package risk indicators?</h2>
+<p>The Tidelift Subscription provides deeper, more meaningful insights that allow you to evaluate latent risk indicators such as package maintenance and end-of-life status, evaluating code contributors and security measures such as two-factor-authentication to eliminate malicious code injections, and more.</p>
```
Comment (katzj) [inline app/views/projects/index.html.erb:97]: Probably want a link here?

**Review by tradiff — APPROVED** [review]

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
 </p>
 
-<p>By logging in you accept our <a href='/terms'>terms of service</a> and <a href='/privacy'>privacy policy</a>.</p>
\ No newline at end of file
+<p>By logging in you accept our <a href='/terms'>terms of service</a> and <a href='/privacy'>privacy policy</a>.</p>
+<hr>
+<h2>Why login to Libraries.io?</h2>
+<p>Libraries.io lets you watch packages that you depend on and notifies you when there's a new release. When you follow a package, new releases will show up here in your newsfeed.</p>
+<p>Libraries.io can automatically keep track of all of the packages that your repositories depend upon across many different package managers.</p>
+<p>Data from Libraries.io can easily be integrated into your day-to-day workflows using our API. Get your API key now! Libraries.io will also email you about new versions of your dependencies, if you add or remove a new dependency it will change the notifications settings for that package as soon as you push to your repositories.</p>
```
Comment (katzj) [inline app/views/sessions/new.html.erb:29]:
```suggestion
<ul>
  <li>Libraries.io lets you watch packages that you depend on and notifies you when there's a new release. When you follow a package, new releases will show up here in your newsfeed.</li>
  <li>Data from Libraries.io can easily be integrated into your day-to-day workflows using <a href="/api">our API</a>. Get your API key now!</li>
  <li>Libraries.io can email you about new versions of your dependencies</li>
</ul>
```

**Review by katzj — COMMENTED** [review]
A few small suggestions

File: app/views/projects/index.html.erb (hunk 2 above)
Comment (bee64) [reply app/views/projects/index.html.erb]: yes!

**Review by katzj — APPROVED** [review]

**Pull request merged**